nettacker/core/messages.py

```
@@ -20,7 +21,8 @@ def application_language():
 
 
 def load_yaml(filename):
-    return yaml.load(StringIO(open(filename, "r").read()), Loader=yaml.FullLoader)
+    yaml_text = Path(filename).read_text(encoding="utf-8")
+    return yaml.load(StringIO(yaml_text), Loader=yaml.FullLoader)
```

Ruff (0.15.2)

[error] Probable use of unsafe loader `FullLoader` with `yaml.load`. Allows instantiation of arbitrary objects. Consider `yaml.safe_load`.

(S506)